examples/java/.gitignore

```diff
@@ -0,0 +1,13 @@
+
+build/
+
+.gradle/
+gradle/
+gradlew
+gradlew.bat
+
+**/.idea/**/workspace.xml
```

Comment:
@chadwalters and I experimented with a number of different styles to get this to work on Windows. This was the one we found that worked to ignore any workspace.xml under an .idea directory potentially in a subdirectory on Windows. The best we can figure is that the implementation of `fnmatch` that Windows Git uses differs from what's documented in the Git manual. Or, we could have misinterpreted the manual. :-) 

If you have a more standard one that works on Windows, I'd be interested in seeing it.